docs/docker-image-build.md

```diff
@@ -0,0 +1,29 @@
+# Build a `ks` docker image
```

Comment:
@GuessWhoSamFoo What do you think about this text?

Reply:
Nit: Referencing Docker outside the context of a CLI should be upper case.